docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+resolve DNS queries that are not for Landrush managed hostnames. If your network
+blocks access to the Google DNS servers, due to firewalls, VPN and so on, as a workaround,
+you need to set a custom upstream DNS server. This server is configured with the
+`config.landrush.upstream` option.
+
+For example, to use 10.25.35.4 as a DNS server, add this line to your `Vagrantfile`:
+
+`config.landrush.upstream '10.25.35.4'`
+
+You will need to check your network configuration on the host or ask your network
+administrator for the right DNS server address to use.
+
+For further details, refer to the
+https://github.com/vagrant-landrush/landrush#guest-is-unable-to-access-the-internet[Guest is unable to access the Internet]
+and the https://github.com/vagrant-landrush/landrush#unmatched-queries[Unmatched Queries]
+sections in the Landrush documentation.
+
+For future reference you can track this https://github.com/projectatomic/adb-atomic-developer-bundle/issues/523[issue].
+
+==== I cannot access OpenShift console from my browser.
+Landrush enables you to access services running in the ADB,  such as OpenShift,
+from the browser. When you start ADB, it provides Landrush with the IP address
+and hostname it wishes to use. Landrush always uses the latest information available.
+Occasionally, using multiple copies of ADB on the same host or other problems with
+ADB can cause Landrush to store stale data in the `hosts.json` file, which prevents
+access to the OpenShift console. +
+
+There are two ways you can remove these stale entries, you can do the following:
+
+. Use `vagrant landrush ls` to check the existing domains for Landrush.
+. Use `vagrant  landrush del <domain_name>` to delete the unwanted domains.
+
+Alternatively, you can:
+
+. Run `vagrant destroy` to destroy the ADB vagrant box.
++
+WARNING: Using vagrant destroy will destroy any data you stored in the Vagrant box.
+You will not be able to restart this instance and will have to create a new one using `vagrant up`.
+
+. Delete extra entries from the `hosts.json` file :
+
+  * Windows users will find this file in: `C:/Users/<USER>/.vagrant.d/data/landrush/hosts.json`
+  * Linux and OS X users will find this file in: `~/.vagrant.d/data/landrush/hosts.json`
++
+The `hosts.json` file should now have an empty list of hostnames, represented by
+a file containing only curly braces `{}`.
+
+To verify that your hostname has been correctly registered with Landrush, run
+`vagrant up`. The expected output is:
+
+....
+$ vagrant.exe landrush ls
+cdk.vm                         172.28.128.3
+3.128.28.172.in-addr.arpa      cdk.vm
+cdk                            cdk.vm
+....
+
+For future reference you can track this https://github.com/projectatomic/adb-atomic-developer-bundle/issues/525[issue].
+
+==== OpenShift Web console fails to reconnect when you switch networks or reconnect to VPN.
+Linux users are most likely to encounter this problem because **NetworkManager**
```

Comment:
I put a blank line between the heading and the text. Makes it easier to read/write (applies to the whole document)